torch/csrc/api/include/torch/nn/functional/linear.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <torch/nn/options/activation.h>
+#include <torch/types.h>
+
+namespace torch {
+namespace nn {
+namespace functional {
+
+inline Tensor linear(const Tensor& input, const Tensor& weight,
+                     const Tensor& bias = {}) {
+  if (input.dim() == 2 and bias.defined()) {
```

Comment:
We probably need to change it to
```suggestion
  if (input.dim() == 2 && bias.defined()) {
```